src/sls_core.js

```diff
@@ -28,7 +27,7 @@
  *   Helper function to retrieve a specific shape within a slide deck based on
  *   a string
  *
- * 22/01/23
+ * 27/01/23
```

Comment:
The format for this date seems to change between files, and the date isn't even present in some files. I'm guessing it's some sort of tracking for last change, but it would be good to make it consistent throughout the codebase.

Reply:
Yes! It was mostly to track files uploaded via clasp, I intend to add this via the script at the beginning of the file (only files in the output, not these ones).

Thanks for flagging!